config/default/inject-config-patch.yaml

```diff
@@ -0,0 +1,5 @@
+- op: add
+  path: /spec/template/spec/containers/0/envFrom/-
+  value:
+    configMapRef:
+      name: shared-environment-config
```

Comment:
The `default` kustomization brings together the kustomizations of manager and oauth and therefore can update them both.

There are 2 other alternatives:
1. we can define the common args in both the manager and oauth kustomizations (which sucks, because there'd be 2 places to do the same thing)
2. we can define a new kustomization (`shared-config` or something) that would define this shared configuration and would be used by both the manager and oauth (and then we could add it to the deployments straight away without using the patch).

Since we never deploy manager and oauth alone, I thought putting this logic in default is good enough, but I can see having a new kustomize for just the shared config used by the manager and oauth deployments should work better if we decide to make it possible to deploy the manager and oauth service separately.

Reply:
I tried approach 3 from above and it actually doesn't work - when building default, which includes both manager and oauth, the shared-config would be defined twice, which kustomize doesn't like. So we're stuck with the aproach as is - patching the deployments of oauth service and manager in the default kustomization.